website/src/components/sidebar.tsx

```diff
@@ -14,32 +14,39 @@ import * as css from "./sidebar.css"
 // Renders a sidebar on the left side of the screen containing a drawer.
 export const Sidebar = () => {
   const dialog = useDialog()
+  // State variable that checks whether this component has loaded in.
+  const [loaded, setLoaded] = useState(false)
 
-  // On load, make sure drawer appears for desktop screens.
+  // On load, make sure drawer is initially visible and is non-modal for desktop screens.
   useEffect(() => {
     dialog.setVisible(true)
+    dialog.setModal(false)
```

Comment:
You can pass these values into `useDialog()` or `useDialogState()` (I don't remember which) instead of in a `useEffect` afterward